### ISSUE TITLE
BCP0113 is missing?

### ISSUE THREAD

**Issue by strogonoff** [relaton/relaton-ietf#104]
This is 404: https://github.com/ietf-tools/relaton-data-rfcsubseries/blob/main/data/BCP0113.yaml

However, this exists: https://github.com/ietf-tools/bibxml-data-archive/blob/main/bibxml9/reference.BCP.0113.xml

This means BibXML service URL /public/rfc/bibxml9/reference.BCP.0113.xml uses fallback.

Maybe this was raised before, let me know if so…

cc @ronaldtse @andrew2net

**Comment by andrew2net** [comment]
@strogonoff the source dataset for relaton-data-rfcsubseries is https://www.rfc-editor.org/rfc-index.xml and it doesn't contain any document with BCP0113 identifier

**Comment by strogonoff** [comment]
Hmm. Should we raise an issue with IETF @ronaldtse?

**Comment by ronaldtse** [comment]
@strogonoff No, we have been here before -- not all RFC numbers or BCP numbers are assigned.

**Comment by ronaldtse** [comment]
@strogonoff @andrew2net yes you are right here. There is no BCP0113 in `rfc-index.xml`. I will contact the RPC.

**Comment by ronaldtse** [comment]
Raised.
